resources/lib/updater.py

```diff
@@ -76,6 +76,10 @@ def GetCurrentUpdateOperation( self, force = False ):
 			self.logger.info( 'Update disabled since database not available' )
 			return 0
 
+		elif force == True:
+			# always full update when force is given by cli-argument
+			return self._getNextUpdateOperation( force )
+
```

Comment:
This change is not really needed, since `_getNextUpdateOperation` is invoked in any case when `force` is `True`